buildpipeline/DotNet-CoreRT-Mac.json

```diff
@@ -293,7 +257,7 @@
       "allowOverride": true
     },
     "Label": {
-      "value": "$(Build.BuildNumber)",
+      "value": "corert-alpha-$(Build.BuildNumber)",
```

Comment:
I see "corert-alpha-" repeated a few times; could this be pulled out into a build variable?